File: benchmark/code-index/datasets/small-synth-repo/py/math.py
def add(a: int, b: int) -> int:
	return a + b


def fib(n: int) -> int:
	if n <= 1:
		return n
	return fib(n - 1) + fib(n - 2)


class Calculator:
	def sum10(self, n: int) -> int:
		return add(n, 10)

File: benchmark/code-index/datasets/small-synth-repo/ts/math.ts
export function add(a: number, b: number): number {
	return a + b
}

export function sub(a: number, b: number): number {
	return a - b
}

export function fib(n: number): number {
	if (n <= 1) return n
	return fib(n - 1) + fib(n - 2)
}

export function isEven(n: number): boolean {
	return n % 2 === 0
}
